src/tests/actions/userActions.test.js

```diff
@@ -89,4 +96,66 @@ describe('userAction', () => {
       },
     ]);
   });
+
+  it('should set get profile initiated to true', async () => {
+    getProfile()(store.dispatch);
+    expect(store.getActions()).toEqual(
+      [
+        { type: GET_PROFILE_INITIATED, payload: true },
+      ],
+    );
+  });
+
+  it('should return user profile details', async () => {
```

Comment:
This as well. I have seen `async` being used in conjunction with `await`  hence my opinion on the way you are using it.

Reply:
I put the async just in case I need to use await to wait for the results to be retrieved before continuation of execution. I shall remove it since I didnt use it anyway.